apps/app-article/src/article-service.js

```diff
@@ -11,7 +11,7 @@ const articleApi = {
 
     },
     getLatestArticles() {
-        return fetch(config.apiUrl + "latest?start=0&limit=5", {
+        return fetch(config.apiUrl + "mostread?start=0&limit=5", {
```

Comment:
Is this right? I mean, the function is called `getLatestArticles`, but we fetch the `mostread` endpoint